jobs/validate.yaml

```diff
@@ -80,8 +77,16 @@
           hour: '12' # Noon
           dow: '2,4,6'  # TuThSa
     charm-channel:
-      - edge
+      - edge:
+          snap_versions: !!python/list 
+            - 1.27/edge
+            - 1.26/edge
+            - 1.25/edge
       - stable:
+          snap_versions: !!python/list 
+            - 1.26/stable
+            - 1.25/stable
+            - 1.24/stable
           dow: '0'      # Sunday
```

Comment:
This makes use of [Variable object references in jjb](https://jenkins-job-builder.readthedocs.io/en/latest/definition.html#variable-references)